### ISSUE TITLE
Internationalization API is not injected with locale

### ISSUE THREAD

**Issue by barjin** [apify/fingerprint-suite#4]
**Describe the bug**
The ECMAScript Internationalization API is not affected by the injected fingerprint's `locale`.

**To Reproduce**
See [@mnmkng 's gist](https://gist.github.com/mnmkng/128cd0fe6594081ecad0bb8dbc8f7b40).

**Expected behavior**
The `Intl` methods should return values consistent with the injected `locale`.

**System information:**
 - OS: [Linux Mint 20.2 Cinnamon]
 - Node.js version [16.15.1]

**Additional context**
Originally brought up by @mnmkng in Slack.

**Comment by petrpatek** [comment]
We need to use the locale context option in playwright: 
```javascript
// Emulate locale and time
const context = await browser.newContext({
  locale: 'de-DE',
  timezoneId: 'Europe/Berlin',
});
```
from what I have found there are two ways to set the accept-language header.
1. By extra headers API
```javascript
await page.setExtraHTTPHeaders({
    'Accept-Language': 'bn'
});
```
2. By a launch argument
```javascript
const browser = await puppeteer.launch({
    headless: false,
    args: ['--lang=bn-BD,bn']
});
```
According to the current implementation, we can only manipulate the fingerprint on a page basis to limit the number of browsers that need to be launched to use a unique "identity", so I would go with the header override method.

**Comment by mnmkng** [comment]
I think the header was being set correctly yesterday when I tried it, but I might've been wrong.

**Comment by barjin** [comment]
The `Accept-language` header is indeed getting set correctly ([here](https://github.com/apify/fingerprint-suite/blob/c7feccca8eeedc713acdb64f97cbcef05639617c/packages/fingerprint-injector/src/fingerprint-injector.ts#L46), basically the first option from @petrpatek 's message). 

The linked PR (#5) fixes the Internationalization API problem from @mnmkng 's gist, but for some reason(???) doesn't work in Firefox. Help, support and ideas greatly appreciated!

**Comment by B4nan** [comment]
Closed via #5
